tarodb/assets_store.go

```diff
@@ -1264,12 +1264,9 @@ func (a *AssetStore) LogPendingParcel(ctx context.Context,
 		// Next, we'll insert the new transaction that anchors the new
 		// anchor point (commits to the set of new outputs).
 		txnID, err := q.UpsertChainTx(ctx, ChainTx{
-			Txid:  newAnchorTXID[:],
-			RawTx: anchorTxBytes,
-			HeightHint: sql.NullInt32{
-				Int32: int32(anchorTxHeightHint),
-				Valid: true,
-			},
+			Txid:       newAnchorTXID[:],
+			RawTx:      anchorTxBytes,
+			HeightHint: sqlInt32(int32(anchorTxHeightHint)),
```

Comment:
So you don't need to cast the integer here, the type inferencing should just work otherwise.